test/functional/p2p_addr_relay.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Test addr relay
+"""
+
+from test_framework.messages import (
+    CAddress,
+    NODE_NETWORK,
+    msg_addr,
+)
+from test_framework.mininode import (
+    P2PInterface,
+)
+from test_framework.test_framework import PivxTestFramework
+from test_framework.util import (
+    assert_equal,
+)
+import time
```

Comment:
nit: `import time` should be above any local imports and separated with an empty line

Reply:
added, will be good to add the rule to the python-lint script.